library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -3079,3 +3082,44 @@ impl<T, const N: usize> From<[T; N]> for VecDeque<T> {
         deq
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature = "kani", issue = "none")]
+mod verify {
+    use core::kani;
+    use crate::collections::VecDeque;
+
+    #[kani::proof]
+    // Stubbing is currently not possible for functions returning pointers as 
+    // Arbitrary trait hasn't been implemented for them
+    // #[kani::stub_verified(<*mut u32>::add)]
+    fn check_vecdeque_swap() {
+        const ARRAY_LEN: usize = 40;
```

Comment:
Should we unify the ARRAY_LEN in all proofs and add comment to it?

Reply:
We can. what should we use?